src/ngx_http_lua_sleep.c

```diff
@@ -71,7 +71,6 @@ ngx_http_lua_ngx_sleep(lua_State *L)
 #ifdef HAVE_POSTED_DELAYED_EVENTS_PATCH
         dd("posting 0 sec sleep event to head of delayed queue");
 
-        coctx->sleep.delayed = 1;
```

Comment:
I remember this is useful, @dndx can you help to confirm it?

Reply:
Yes, this flag is used by the `HAVE_POSTED_DELAYED_EVENTS_PATCH` to signal that the event should be executed the next iteration. Without it the sleep will be executed immediately in the current iteration.

Reply:
I know, but the next iteration will have one more epoll_wait(0) system call

Reply:
we need to make sure the behavior is expected.